vaxrank/epitope_prediction.py

```diff
@@ -109,6 +88,24 @@ def index_contains_kmer(fm, kmer):
         break
     return found
 
+def generate_protein_sequences(genome):
```

Comment:
If I test that fix alone without adjusting the mhctools code then tests will fail due to `source_sequence`. Conversely, if I fix the mhctools issue first then tests fail in Py2 due to shellinford. Is it a good idea to merge a PR with failing tests in anticipation of fixing in a second related PR? Or more generally, what do you do when fixing two bugs simultaneously?